packages/@lwc/compiler/package.json

```diff
@@ -21,10 +21,10 @@
         "@lwc/shared": "1.5.1",
         "@lwc/style-compiler": "1.5.1",
         "@lwc/template-compiler": "1.5.1",
-        "@rollup/plugin-replace": "^2.3.1",
-        "babel-preset-compat": "^0.22.0",
+        "@rollup/plugin-replace": "~2.3.2",
+        "babel-preset-compat": "~0.22.0",
         "rollup": "^1.32.1",
```

Comment:
```suggestion
        "rollup": "~1.32.1",
```

Reply:
That's on my todo list. Wanted to update it in a separate PR.